### ISSUE TITLE
ERROR: FaceDetector is undefined

### ISSUE THREAD

**Issue by armand1m** [mutebg/azure-face-ui#3]
First of all, thanks for contributing to open source. This project is amazing and helped me a lot through understanding the Azure Face API.

However, the Face route has some issues hidden:

In the componentDidMount life cycle method, when trying to run this code:

```js
const faceDetector = new FaceDetector({
  fastMode: true,
  maxDetectedFaces: 1
});
```

it throws a `FaceDetector is undefined` error in the console, and since this is inside of a interval, it keeps throwing this error.

However, since this is inside a `try { //code } catch (e) { //nothing }`, it keeps hidden until you debug it.

I'm struggling to find this reference, from where did you take this FaceDetector class?

**Comment by armand1m** [comment]
I suppose you're using this https://paul.kinlan.me/face-detection/ right?

**Comment by mutebg** [comment]
Hi @armand1m you are right.
I am using face-detection which is Browser API under a flag at Chrome Canary ( and probably will never be officially added to other browsers ). So my goal is to replace it with https://websightjs.com/ which is WebAssembly ( supported in all modern browsers )

Hope next week will have more time to work on that and finally make a stable version of it :)

Cheers

**Comment by armand1m** [comment]
Just opened a pull request to show an error alert when this feature is not available. In the future, I'll implement websight js to this project for private purposes, and may open a PR for you with the feature implemented. Thanks for contributing to OSS!

**Comment by mutebg** [comment]
thanks for your contribution, 
I also investigated websight.js  but didn't have time yet to implement it. 
PR's are welcome so let me know, will be happy to use this outside chrome canary.

**Comment by armand1m** [comment]
Alert implemented in #5
